src/neuroconv/tools/hdmf.py

```diff
@@ -90,3 +95,42 @@ def _get_maxshape(self) -> tuple:
 
     def _get_data(self, selection: Tuple[slice]) -> np.ndarray:
         return self.data[selection]
+
+    def is_pickleable(self) -> bool:
+        if isinstance(self.data, (np.memmap, np.array)):
```

Comment:
Will need to remove `np.array` here; easy support is only offered for memory-map like lazy access to source data